products/sle/main.pm

```diff
@@ -659,6 +659,10 @@ testapi::set_distribution(susedistribution->new());
 if (is_kernel_test()) {
     load_kernel_tests();
 }
+elsif (get_var('NVMFTESTS')) {
+    boot_hdd_image();
+    load_nvmftests();
+}
```

Comment:
well, this might fix it for you but please find a solution that prevents further duplication in openSUSE+SLE

Reply:
removed this change in order to fix the test itself for now